src/nodes/core/Node.js

```diff
@@ -531,9 +532,9 @@ class Node extends EventDispatcher {
 	/**
 	 * The method can be implemented to update the node's internal state before it is used to render an object.
 	 * The {@link Node#updateBeforeType} property defines how often the update is executed.
+	 * This method should return an optional bool that indicates whether the implementation actually performed an update or not (e.g. due to caching).
 	 *
 	 * @param {NodeFrame} frame - A reference to the current node frame.
-	 * @return {Boolean?} An optional bool that indicates whether the implementation actually performed an update or not (e.g. due to caching).
```

Comment:
Even though the node does not return a bool, `@return` should define the type and the description for this abstract method.

Reply:
This also ends up throwing an eslint error as there is no direct return in this abstract function, that's why I move the annotation to a standard description sentence.

Reply:
Settings the @abstract flag seems to fix the issue